src/components/cart/CartToggle/CartToggle.tsx

```diff
@@ -8,9 +9,12 @@ function CartToggle() {
   const { totalUniqueItems } = useCart()
 
   return (
-    <Button {...btnProps} className="mx-3">
-      cart: {totalUniqueItems}
-    </Button>
+    <>
+      <Button {...btnProps} className="mx-3">
+        cart: {totalUniqueItems}
+      </Button>
+      <Link to="http://chk.vtex-base1.tk/checkout">Go to checkout</Link>
```

Comment:
We already have this. Take a look at: https://github.com/vtex-sites/base.store/blob/9f1bbc6f593bbc40827ff6bc3e9de62988dd02bf/src/components/cart/CartSidebar/CartSidebar.tsx#L38